ocean_lib/models/data_token.py

```diff
@@ -252,9 +251,8 @@ def get_all_transfers_from_events(
     def get_transfer_event(self, block_number, sender, receiver):
```

Comment:
Please make `get_transfer_event` call `get_event_logs("Transfer", ...)`.  The rest of the logic about asserting only a single event occurred is still okay.